components/Challenges.js

```diff
@@ -31,6 +36,14 @@ export default function ChallengeList({ challenges, onDelete }) {
             <h2>{expandedChallenge === challenge.id ? "▼" : "▶︎"}</h2>
             <h2>{challenge.title}</h2>
 
+            <Checkbox
+              type="checkbox"
+              id="done"
+              name="done"
+              value="done"
+              checked={challenge.done}
+            ></Checkbox>
```

Comment:
You can make elements, that do not have any children, self-closing :)
```suggestion
            <Checkbox
              type="checkbox"
              id="done"
              name="done"
              value="done"
              checked={challenge.done}
            />
```